modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -70,8 +70,27 @@ platform:
 publish: External
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed12345 AAAA...
+compute:
```

Comment:
This is probably more of a question for the SME and QE reviewers, but is it ok to have two top level `compute: platform` sections in this YAML configuration? Or, should this content be included in the existing top level `compute` section that is towards the top of this example?

Reply:
@Darth-Mera  - can you answer this?

Reply:
And lines 73-84 should be deleted